coalib/coala_delete_orig.py

```diff
@@ -15,8 +16,9 @@ def main(log_printer=None, section: Section=None):
     if start_path is None:
         return 255
 
-    orig_files = Globbing.glob(os.path.abspath(
-        os.path.join(start_path, '**', '*.orig')))
+    # start_path may have unintended glob characters
+    orig_files = Globbing.glob(os.path.join(
+        glob_escape(start_path), '**', '*.orig'))
```

Comment:
It's actually unnecessary since `get_config_directory` always returns a full path (since `os.path.getcwd()`, `str(section["config"])` are absolute paths which are the only return values)